lumicks/pylake/tests/test_channels/test_channels.py

```diff
@@ -116,23 +116,51 @@ def test_calibration_continuous_channels():
 
 
 def test_slice_properties():
-    size = 5
-    s = channel.Slice(channel.TimeSeries(np.random.rand(size), np.random.rand(size)))
-    assert len(s) == size
-    assert s.sample_rate is None
+    size = 10
 
     s = channel.Slice(channel.Continuous(np.random.rand(size), start=0, dt=1))
     assert len(s) == size
     assert s.sample_rate == 1e9
+    timesteps = np.asarray([1], dtype=int)
+    np.testing.assert_equal(s._timesteps, timesteps)
+    np.testing.assert_equal(s.sample_rates, 1e9 / timesteps)
+
+    rng = np.random.default_rng()
+    timestamps = np.unique(np.sort(rng.integers(1e9, 10e9, size=size)))
```

Comment:
I would move these lines up to the top (right under `size = 10`) to keep all the declaration stuff together at the front; I think it's a bit more organized that way

Reply:
Done